WORKSPACE

```diff
@@ -122,25 +118,27 @@ load("//dependencies/vaticle:artifacts.bzl", "vaticle_typedb_artifact", "vaticle
 vaticle_typedb_artifact()
 vaticle_typedb_cluster_artifact()
 
-# Load maven
-load("@vaticle_typedb_common//dependencies/maven:artifacts.bzl", vaticle_typedb_common_artifacts = "artifacts")
-load("@vaticle_typeql//dependencies/maven:artifacts.bzl", vaticle_typeql_artifacts = "artifacts")
-load("@vaticle_factory_tracing//dependencies/maven:artifacts.bzl", vaticle_factory_tracing_artifacts = "artifacts")
-load("//dependencies/maven:artifacts.bzl", vaticle_typedb_client_java_artifacts = "artifacts", vaticle_typedb_client_java_overrides = "overrides")
-
 ###############
 # Load @maven #
 ###############
 
+# Load maven artifacts
+load("@vaticle_dependencies//tool/common:deps.bzl", vaticle_dependencies_tool_maven_artifacts = "maven_artifacts")
+load("@vaticle_typedb_common//dependencies/maven:artifacts.bzl", vaticle_typedb_common_maven_artifacts = "artifacts")
+load("@vaticle_typeql//dependencies/maven:artifacts.bzl", vaticle_typeql_maven_artifacts = "artifacts")
+load("@vaticle_factory_tracing//dependencies/maven:artifacts.bzl", vaticle_factory_tracing_maven_artifacts = "artifacts")
+load("//dependencies/maven:artifacts.bzl", vaticle_typedb_client_java_maven_artifacts = "artifacts", vaticle_typedb_client_java_maven_overrides = "overrides")
+load("@vaticle_bazel_distribution//maven:deps.bzl", vaticle_bazel_distribution_maven_artifacts = "maven_artifacts")
+
 load("@vaticle_dependencies//library/maven:rules.bzl", "maven")
 maven(
-    vaticle_factory_tracing_artifacts +
-    vaticle_typedb_common_artifacts +
-    vaticle_typeql_artifacts +
+    vaticle_factory_tracing_maven_artifacts +
```

Comment:
The aliases were previously inconsistent in their naming. Now, all of them end in `maven_artifacts` (except overrides).